### PR TITLE
added cob4_homing package

### PR DESCRIPTION
**Pull request by ipa-nhg**

### CODE REVIEW

**Comment by ipa-nhg** [comment]
@ipa-mdl is it ok for you add the cob4_homming package to setup repository?

File: cob4_homing/package.xml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0"?>
+<package>
+  <name>cob4_homing</name>
+  <version>0.0.0</version>
+  <description>The cob4_homing package</description>
+
+  <!-- One maintainer tag required, multiple allowed, one person per tag --> 
+  <!-- Example:  -->
+  <!-- <maintainer email="jane.doe@example.com">Jane Doe</maintainer> -->
+  <maintainer email="mdl@todo.todo">mdl</maintainer>
```
Comment (ipa-mdl) [inline cob4_homing/package.xml]: mathias.luedtke@ipa.fraunhofer.de

File: cob4_homing/package.xml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+
+  <!-- One license tag required, multiple allowed, one license per tag -->
+  <!-- Commonly used license strings: -->
+  <!--   BSD, MIT, Boost Software License, GPLv2, GPLv3, LGPLv2.1, LGPLv3 -->
+  <license>TODO</license>
+
```
Comment (ipa-mdl) [inline cob4_homing/package.xml]: TODO..

File: cob4_homing/package.xml (hunk 1 above)
Comment (ipa-nhg) [inline cob4_homing/package.xml]: Done

File: cob4_homing/package.xml (hunk 2 above)
Comment (ipa-nhg) [inline cob4_homing/package.xml]: Done

**Comment by ipa-fmw** [comment]
FR: OK

**Pull request merged**